tests/blackbox/testlib/utils.py

```diff
@@ -53,16 +56,19 @@ def random_string(length=4):
 
 def process_exists(name):
     """
-    Walk the process table looking for executable 'name', returns pid if one
-    found, else return None
+    Look through processes, using their pids, to find one matching 'name'.
+    Return None if no such process found, else return the pid.
+    :param name: name of process to check
+    :type name: string
+    :return: None or pid
```

Comment:
You can also use the ```rtype``` markup, which should say something like: ```NoneType or int```

Reply:
Is this in addition to `return`, or instead of it?

Reply:
In addition: ```:param``` is to ```:type``` as ```:return``` is to ```:rtype".